bylaws.tex

```diff
@@ -358,4 +358,14 @@
 	\item Obtain greater than or equal to 60\% (rounded up to the nearest whole person) of signatures in the Introductory Packet or successfully complete Introductory Evaluations.
 \end{enumerate}
 
+\bsection{Code of Conduct}
+The Code of Conduct located at \href{https://github.com/ComputerScienceHouse/CodeOfConduct} is the canonical Code of Conduct for Computer Science House accounts.
+\\* \\*
+Members are bound to the Code of Conduct revision that they sign when initially creating their account. Members may sign a more recent revision of the Code of Conduct to update their agreement.
+\bsubsection{Changes}
+The following process is used for making changes to the Code of Conduct document.
+\begin{enumerate}
+	\item A Root Type Person drafts a change to the Code of Conduct and makes the difference file of the change as well as the summary of the change publicly available.
```

Comment:
The 'as well as' here makes this sentence awkward.

A Root Type Person drafts a change to the Code of Conduct and makes publicly available both the summary and difference file of the change.